parameterized/parameterized.py

```diff
@@ -19,9 +19,14 @@
     class SkipTest(Exception):
         pass
 
+try:
+    import pytest
+except ImportError:
+    pytest = None
+
 PY3 = sys.version_info[0] == 3
 PY2 = sys.version_info[0] == 2
-
+PYTEST4 = pytest and pytest.__version__ >= '4.0.0'
```

Comment:
This doesn't do the sames as the `PY2` and `PY3` constants above. It also evaluates to `True` if the runner is nose or unittest but when pytest is importable.

Workaround: add `detect_runner() == 'pytest'` to any conditional using it.